lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
We could, but does it buy anything?

Viewer has [similar init code](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-viewer/app/src/main.js#L44) and uses a main.js. But the only reason I put that a separate script was because the code uses `require()`.